site/includes/social-media.hbs

```diff
@@ -0,0 +1,31 @@
+<div class="provisional followus followus-vertical">
```

Comment:
Use a `<section>` instead of a `<div>`

Reply:
It is currently coded like depicted in the DTO-TBS prototype. The parent node is a <section> with two classes identified.  Would you like me to remove this parent and move to the include component?

Reply:
This has been done.